src/dialect/database-introspector.ts

```diff
@@ -26,6 +26,11 @@ export interface DatabaseMetadataOptions {
    * such as the migration tables.
    */
   withInternalKyselyTables: boolean
+
+  /**
+   * If this is true, the metadata contains the foreign tables.
+   */
+  withForeignTables?: boolean
```

Comment:
Why is this under a flag? Why not always grab them?

Reply:
In-order to not change existing deployments. I figure they may be side effects so rolling it out like this would prevent that.

Reply:
> In-order to not change existing deployments. I figure they may be side effects so rolling it out like this would prevent that.

These changes would end up in the next minor version. You'd have to manually bump, knowing there might be breaking changes.

There's valuable information in "side effects" about how people are using introspection that will be lost.

Filtering, or allowing to filter, such tables, is a downstream problem that's trivial to solve.

Let's always include such tables, and react to any issues that emerge.